packages/components/src/components/hds/alert/index.ts

```diff
@@ -8,34 +8,37 @@ import { action } from '@ember/object';
 import { assert } from '@ember/debug';
 import { guidFor } from '@ember/object/internals';
 import { tracked } from '@glimmer/tracking';
+import {
+  HdsAlertTypeValues,
+  HdsAlertColorValues,
+  type HdsAlertSignature,
```

Comment:
a note of caution even though this is valid way to import types but https://github.com/embroider-build/addon-blueprint/issues/272#issuecomment-1992512789

Reply:
@alex-ju @natmegs 👆🏻